src/Bladeburner/Actions/ActionIdentifier.ts

```diff
@@ -0,0 +1,66 @@
+import type { ActionIdentifier } from "../Types";
+import { BladeburnerActionType } from "@enums";
+import { assertLoadingType } from "../../utils/TypeAssertion";
+import { getEnumHelper } from "../../utils/EnumHelper";
+
+/** Loads an action identifier
+ * This is used for loading ActionIdentifier class objects from pre-2.6.1
+ * Should load both the old format and the new format */
+export function resolveActionIdentifier(identifier: unknown): ActionIdentifier | null {
+  if (!identifier || typeof identifier !== "object") return null;
+  assertLoadingType<ActionIdentifier>(identifier);
+  for (const matcher of TypeToActionMatchers) {
+    const id = matcher(identifier.name);
+    if (id) {
+      return id;
+    }
+  }
+
+  return null;
+}
+
+/** These shorthands match those documented in the BB Terminal Help */
+export const TerminalShorthands = {
+  [BladeburnerActionType.Contract]: <readonly string[]>["contract", "contracts", "contr"],
```

Comment:
You shouldn't need `<readonly string[]>` here anymore, it's implied by `as const`.